build/charts/theia/provisioning/dashboards/service_dependency_graph_dashboard.json

```diff
@@ -0,0 +1,220 @@
+{
+  "annotations": {
+    "list": [
+      {
+        "builtIn": 1,
+        "datasource": "-- Grafana --",
+        "enable": true,
+        "hide": true,
+        "iconColor": "rgba(0, 211, 255, 1)",
+        "name": "Annotations & Alerts",
+        "target": {
+          "limit": 100,
+          "matchAny": false,
+          "tags": [],
+          "type": "dashboard"
+        },
+        "type": "dashboard"
+      }
+    ]
+  },
+  "editable": true,
+  "fiscalYearStartMonth": 0,
+  "graphTooltip": 0,
+  "id": 1,
+  "iteration": 1669268194538,
+  "links": [],
+  "liveNow": false,
+  "panels": [
+    {
+      "gridPos": {
+        "h": 13,
+        "w": 14,
+        "x": 0,
+        "y": 0
+      },
+      "id": 2,
+      "options": {
+        "seriesCountSize": "sm",
+        "showSeriesCount": false,
+        "text": "Default value of text input option"
+      },
+      "targets": [
+        {
+          "builderOptions": {
+            "database": "",
+            "fields": [
+              "sourcePodName",
+              "sourcePodLabel",
+              "sourceNodeName",
+              "destinationPodName",
+              "destinationPodLabel",
+              "destinationNodeName",
+              "destinationServicePortName",
+              "octetDeltaCount"
+            ],
+            "filters": [
+              {
+                "condition": "AND",
+                "filterType": "custom",
+                "key": "destinationPodName",
+                "operator": "IS NOT NULL",
+                "type": "",
+                "value": []
+              },
+              {
+                "condition": "AND",
+                "filterType": "custom",
+                "key": "sourcePodName",
+                "operator": "IN",
+                "type": "string",
+                "value": [
+                  "${queryPodName:raw}"
+                ]
+              }
+            ],
+            "mode": "list",
+            "table": "flows"
+          },
+          "datasource": {
+            "type": "grafana-clickhouse-datasource",
+            "uid": "9JmyLZOVk"
```

Comment:
In testing it was the same, but it may have changed after I deleted the grafana-clickhouse-datasource folder. I'll manually change it back to the original.